src/runway/module/staticsite.py

```diff
@@ -27,7 +27,9 @@ class StaticSite(RunwayModule):
 
     def setup_website_module(self, command):
         """Create stacker configuration for website module."""
-        name = self.options.get('name', self.options.get('path'))
+        original_name = self.options.get('name', self.options.get('path'))
+        # Make sure we remove any `.static` extension
+        name = original_name.split('.')[0]
```

Comment:
I can't think of any specific conflicts / edge cases at the moment, but I think it'd be better to codify the exact intent here. Something like:

```
name = self.options.get('name', self.options.get('path'))
directory_suffix = '.static'
if name.endswith(directory_suffix):
    name = name[:-len(directory_suffix)]
```